### DISCUSSION TITLE
Query related to shop billing charges

### DISCUSSION THREAD

**Discussion by yasir-naseer** [osiset/laravel-shopify#925] [Q&A]
I am using billing to charge the shops based on monthly plans, when the store install the app and subscribe to a plan, it creates an charge against the shop in the charges table, now i want to know that after the month, will the second charge will be created by the app automatically in the charges table or do i have to make create it using webhook or other methods. kindly guide about this

**Comment by gnikyt** [comment]
Shopify charges in the background on their side, nothing gets updated for charges as the data stays the same.
